src/browser-bookmarks/background/import-bookmarks.js

```diff
@@ -0,0 +1,256 @@
+// Imports the full browser's bookmarks into our database.
+// The browser's bookmarkItems and visitItems are quite straightforwardly
+// converted to pageDocs and visitDocs (sorry for the confusingly similar name).
+
+import db from 'src/pouchdb'
+import { updatePageSearchIndex } from 'src/search/find-pages'
+import { isWorthRemembering, generateVisitDocId,
+         visitKeyPrefix, convertVisitDocId } from 'src/activity-logger'
+import { generatePageDocId } from 'src/page-storage'
+
+
+/*
+@Description
+Array getBookmarkTree()
+Get the Whole Browser Bookmark Tree.
+
+@Params
+void
+
+@Return
+Array bookmarkItems
+*/
+async function getBookmarkTree() {
```

Comment:
I think it's better to just stick to using `browser.bookmarks.getRecent()`, as talked about [here](https://github.com/WebMemex/webmemex-extension/issues/29#issuecomment-283826516).

If `getRecentBookmarks` function is fixed up, I think you should replace this (`getBookmarkTree`) with that.